src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/JpaReviewSchedulePersistenceAdapter.kt

```diff
@@ -71,6 +77,30 @@ class JpaReviewSchedulePersistenceAdapter(
       updatedAtPrison = updateReviewScheduleStatusDto.prisonId
     }
 
-    return reviewScheduleEntityMapper.fromEntityToDomain(reviewScheduleRepository.save(reviewScheduleEntity))
+    val savedReviewScheduleEntity = reviewScheduleRepository.save(reviewScheduleEntity)
```

Comment:
I know not added in this PR, but I've just noticed this method sig is not marked as `@Transactional` - do you think we need to / worth doing in this PR? 
```
override fun updateReviewScheduleStatus(....
```